codecov.yml

```
@@ -0,0 +1,15 @@
+      default:
+        target: 10%    # the required coverage value
+        threshold: 1%  # the leniency in hitting the target
+    patch:
+      default:
+        target: 10%
+        threshold: 1%
+
+ignore:
+  - "test/**/*"       # ignore test files
+  - "out/**/*"        # ignore compiled output
+  - "**/*.d.ts"      # ignore type declaration files 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)